### PR TITLE
Branch update delete patient command

### PR DESCRIPTION
**Pull request by wadia82**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeletePatientCommand.java [hunk 1]
```diff
@@ -26,14 +26,14 @@ public class DeleteCommand extends Command {
...
-    public DeleteCommand(Index targetIndex) {
+    public DeletePatientCommand(Index targetIndex) {
         this.targetIndex = targetIndex;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> lastShownList = model.getFilteredPersonList();
+        List<Person> lastShownList = model.getFilteredPatientList();
```
Comment (TZL0) [inline src/main/java/seedu/address/logic/commands/DeletePatientCommand.java:36]: Patient?

File: src/main/java/seedu/address/logic/commands/EditPersonCommand.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public EditPersonCommand(Index index, EditPersonDescriptor editPersonDescriptor)
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> lastShownList = model.getFilteredPersonList();
+        List<Person> lastShownList = model.getFilteredPatientList();
```
Comment (TZL0) [inline src/main/java/seedu/address/logic/commands/EditPersonCommand.java:74]: Patient

File: src/main/java/seedu/address/logic/commands/SelectPersonCommand.java [hunk 3]
```diff
@@ -38,7 +38,7 @@ public SelectPersonCommand(Index index) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> lastShownList = model.getFilteredPersonList();
+        List<Person> lastShownList = model.getFilteredPatientList();
```
Comment (TZL0) [inline src/main/java/seedu/address/logic/commands/SelectPersonCommand.java:41]: Patient?

File: src/main/java/seedu/address/model/Model.java [hunk 4]
```diff
@@ -85,7 +85,7 @@ public interface Model {
     void setPerson(Person target, Person editedPerson);
```
Comment (TZL0) [inline src/main/java/seedu/address/model/Model.java:85]: Patient?

File: src/main/java/seedu/address/model/Model.java [hunk 5]
```diff
@@ -85,7 +85,7 @@ public interface Model {
...
 
     /** Returns an unmodifiable view of the filtered person list */
-    ObservableList<Person> getFilteredPersonList();
+    ObservableList<Person> getFilteredPatientList();
```
Comment (TZL0) [inline src/main/java/seedu/address/model/Model.java:88]: Patient?

File: src/main/java/seedu/address/model/ModelManager.java [hunk 6]
```diff
@@ -151,7 +151,7 @@ public void setAppointment(Appointment target, Appointment editedAppointment) {
      * {@code versionedAddressBook}
      */
     @Override
-    public ObservableList<Person> getFilteredPersonList() {
+    public ObservableList<Person> getFilteredPatientList() {
         return filteredPersons;
```
Comment (TZL0) [inline src/main/java/seedu/address/model/ModelManager.java:155]: Patient?

**Review by TZL0 — APPROVED** [review]

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 7]
```diff
@@ -62,7 +62,7 @@ public ReadOnlyAddressBook getAddressBook() {
 
     @Override
     public ObservableList<Person> getFilteredPersonList() {
```
Comment (Fuzanwenn) [inline src/main/java/seedu/address/logic/LogicManager.java:64]: Better change to getFilteredPatientList() instead.

File: src/main/java/seedu/address/model/Model.java (hunk 4 above)
Comment (Fuzanwenn) [inline src/main/java/seedu/address/model/Model.java:85]: setPatient

File: src/main/java/seedu/address/model/ModelManager.java (hunk 6 above)
Comment (Fuzanwenn) [inline src/main/java/seedu/address/model/ModelManager.java:155]: return filteredPatients

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 8]
```diff
@@ -139,24 +139,24 @@ public static void assertCommandFailure(Command command, Model actualModel, Stri
...
      * {@code model}'s address book.
      */
     public static void showPersonAtIndex(Model model, Index targetIndex) {
-        assertTrue(targetIndex.getZeroBased() < model.getFilteredPersonList().size());
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredPatientList().size());
 
-        Person person = model.getFilteredPersonList().get(targetIndex.getZeroBased());
+        Person person = model.getFilteredPatientList().get(targetIndex.getZeroBased());
         final String[] splitName = person.getName().fullName.split("\\s+");
         model.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
```
Comment (Fuzanwenn) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:157]: updateFilteredPatientList

File: src/test/java/seedu/address/logic/commands/DeletePatientCommandTest.java [hunk 9]
```diff
@@ -20,47 +20,47 @@
...
-        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
+        Person personToDelete = model.getFilteredPatientList().get(INDEX_FIRST_PERSON.getZeroBased());
+        DeletePatientCommand deletePatientCommand = new DeletePatientCommand(INDEX_FIRST_PERSON);
 
-        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_PATIENT_SUCCESS, personToDelete);
+        String expectedMessage = String.format(DeletePatientCommand.MESSAGE_DELETE_PATIENT_SUCCESS, personToDelete);
 
         ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
         expectedModel.deletePerson(personToDelete);
```
Comment (Fuzanwenn) [inline src/test/java/seedu/address/logic/commands/DeletePatientCommandTest.java:37]: deletePatient

File: src/test/java/seedu/address/logic/commands/DeletePatientCommandTest.java [hunk 10]
```diff
@@ -20,47 +20,47 @@
...
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredPatientList().size() + 1);
+        DeletePatientCommand deletePatientCommand = new DeletePatientCommand(outOfBoundIndex);
 
-        assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_PATIENT_DISPLAYED_INDEX);
+        assertCommandFailure(deletePatientCommand, model, Messages.MESSAGE_INVALID_PATIENT_DISPLAYED_INDEX);
     }
 
     @Test
     public void execute_validIndexFilteredList_success() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
```
Comment (Fuzanwenn) [inline src/test/java/seedu/address/logic/commands/DeletePatientCommandTest.java:52]: Patient

**Review by Fuzanwenn — APPROVED** [review]

**Pull request merged**